plugins/modules/monitoring/uptimerobot.py

```diff
@@ -8,138 +8,178 @@
 
 
 DOCUMENTATION = '''
-
 module: uptimerobot
-short_description: Pause and start Uptime Robot monitoring
+short_description: Create, remove, pause and start monitors in Uptimerobot
 description:
-    - This module will let you start and pause Uptime Robot Monitoring
-author: "Nate Kingsley (@nate-kingsley)"
+    - This module lets you create, remove, pause and start monitors in Uptimerobot
+author:
+- "Nate Kingsley (@nate-kingsley)"
+- "Ninjiner (@Ninjiner)"
+version_added: "1.9"
 requirements:
     - Valid Uptime Robot API Key
 options:
     state:
         description:
             - Define whether or not the monitor should be running or paused.
+            - created and absent are added in version 1.2.0
```

Comment:
```suggestion
            - C(created) and C(absent) were added in community.general 1.3.0.
```

Reply:
Changed in https://github.com/ansible-collections/community.general/pull/961/commits/5d5c6bcc163c83624ae1294289a67fdc47fb9029

Reply:
You didn't use `C(...)` and also the wrong case for the options.

Reply:
Changed in https://github.com/ansible-collections/community.general/pull/961/commits/dab1a424d7eb88d6ab69406896be8141a8e78f77